Dockerfile

```diff
@@ -12,12 +12,14 @@ RUN INSTALL_PKGS=" \
     yum clean all && \
     mkdir /tmp/ocp-clo && \
     chmod og+w /tmp/ocp-clo
-COPY --from=builder _output/bin/cluster-logging-operator /usr/bin/
+COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/_output/bin/cluster-logging-operator /usr/bin/
 COPY scripts/* /usr/bin/scripts/
 RUN mkdir -p /usr/share/logging/
 COPY files/ /usr/share/logging/
-COPY manifests/4* /manifests/
-COPY manifests/cluster-logging.package.yaml /manifests/
+
+COPY --from=builder /go/src/github.com/openshift/cluster-logging-operator/manifests /manifests
+RUN rm /manifests/art.yaml
```

Comment:
You can keep the glob since it looks like having a flat directory structure in `/manifests` is the desired behavior. The COPY command can be squeezed into one instruction:

```
COPY manifests/cluster-logging.package.yaml manifests/4* /manifests/
```

Reply:
we actually dont want a flat directory structure

Reply:
then disregard - what you have here makes sense.